client/src/monaco-services.ts

```diff
@@ -31,10 +31,14 @@ export namespace MonacoServices {
             window: new ConsoleWindow()
         }
     }
-    export function install(_monaco: typeof monaco, options: Options = {}): MonacoServices {
+    export function install(_monaco: typeof monaco, options: Options = {}): Disposable {
```

Comment:
You need to import there otherwise the code does not compile:
```javascript
import { DisposableCollection } from './disposable';
import { Disposable } from 'vscode-jsonrpc';
```

Reply:
I didn't know what this part didn't get staged before I commited => d53522ea89007ab50c0d413aa5dfc3fde7f639b0

sorry

Reply:
No worries, I suggested the wrong Disposable in the proposed solution 🙈